src/batch-submitter/batch-submitter.ts

```diff
@@ -25,9 +26,9 @@ export interface Range {
   end: number
 }
 export interface ResubmissionConfig {
-  resubmissionTimeout: number,
-  minGasPriceInGwei: number,
-  maxGasPriceInGwei: number,
+  resubmissionTimeout: number
+  minGasPriceInGwei: number
+  maxGasPriceInGwei: number
```

Comment:
Unrelated to the core functionality implemented here, but there seem to be quite a few stylistic changes made here (like the removal of commas above). We don't have a linting / formatting CI step yet (which we ought to soon hopefully). Absent that could you try to strip them from these commits?